src/CMake/dkms.cmake

```diff
@@ -18,19 +18,19 @@ SET (DKMS_POSTINST "postinst")
 SET (DKMS_PRERM "prerm")
```

Comment:
Copyright date (line 2) needs to be updated.  Being that we are AMD, you might wish to add the following line:
`Copyright (C) 2022 Advanced Micro Devices, Inc.`